src/components/layout/Footer.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styles from "./layout.module.css";
+
+const Footer = () => (
+  <div className={styles.footer}>
+    <div className={styles.textWrapper}>
+      <span>Decred developers | 2016 - 2019</span>
```

Comment:
This "2019" shouldn't be hardcoded. Please use something like: `const year = new Date().getFullYear();`